tests/integration/multicluster/centralremotekubeconfig/main_test.go

```diff
@@ -0,0 +1,142 @@
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package centralremotekubeconfig
+
+import (
+	"context"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/pkg/test/scopes"
+	"istio.io/istio/pkg/test/util/file"
+	"istio.io/istio/tests/integration/multicluster"
+
+	kubeApiCore "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	ist istio.Instance
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Why do we need a special test for central istiod? Can't we just run any multicluster test with different flags? In other words, have a separate test target that we run on prow?

Reply:
Current multi cluster test can not cover this as this is just added in 1.7.  -we need create istio-kubeconfig secret before hand also need to install the remote cluster first.

Reply:
We need to make it so that the test is agnostic to the underlying topology. All of this should be handled by the istio installation internally, not from the test code.

Reply:
@irisdingbj I'm not sure why you marked this conversation as resolved. We don't want central istiod-specific tests. We want to just run all of the Istio tests in central istiod mode (via flags). Please remove these tests.